WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Summary/SimplePaymentsSummary.swift

```diff
@@ -170,16 +170,25 @@ private struct PaymentsSection: View {
 
                 TitleAndToggleRow(title: SimplePaymentsSummary.Localization.chargeTaxes, isOn: $viewModel.enableTaxes)
                     .padding(.horizontal)
+                    .renderedIf(viewModel.showChargeTaxesToggle)
 
                 Group {
                     Text(SimplePaymentsSummary.Localization.taxesDisclaimer)
                         .footnoteStyle()
                         .padding(.horizontal)
                         .fixedSize(horizontal: false, vertical: true)
```

Comment:
I found an edge case where you first have some tax set up in your store, turn on the Charge taxes toggle and then go back to remove the taxes. When creating a new simple payment, you'll still display this text:

<img src="https://user-images.githubusercontent.com/5533851/150723066-5f0aab40-8cbf-4dc8-bb5b-812aa7b90d06.png" width=320 />

Should this text be rendered only if `showChargeTaxesToggle` is true?

Reply:
Great catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5943/commits/41173b8cfcd032818cfa486bb75a895a9abe9101